src/borg/helpers.py

```diff
@@ -756,6 +756,22 @@ def __format__(self, format_spec):
         return format_file_size(int(self)).__format__(format_spec)
 
 
+class OutputTimestamp:
+    def __init__(self, ts: datetime):
+        if ts.tzinfo == timezone.utc:
+            ts = to_localtime(ts)
+        self.ts = ts
+
+    def __format__(self, format_spec):
+        return format_time(self.ts)
+
+    def __str__(self):
+        return '{}'.format(self)
+
+    def to_json(self):
+        return isoformat_time(self.ts)
```

Comment:
somehow slightly misleading name. you use it for json output, but it is not json by itself.
in fact, the only difference is that it is rather optimized to be machine readable (and really iso 8601) and the other format is more human readable and only parts of it are formatted like in iso.